tests/s3/copy_object/test_copy_obj_simultaneous_del.py

```diff
@@ -108,12 +108,30 @@ def create_put_object(self, bucket, obj, size, base='1M'):
         assert_utils.assert_true(resp[0], resp[1])
         return self.s3_obj.put_object(bucket, obj, self.file_path)
 
+    @staticmethod
+    def head_object_wrapper(bucket, key, exception=None):
+        """Head object wrapper"""
+        s3_obj = s3_test_lib.S3TestLib()
+        try:
+            return s3_obj.object_info(bucket, key)
+        except CTException as err:
+            LOGGER.info("Exception in copy object %s", err)
+            if exception:
+                return exception in err.message, f"Expected {exception} Received {err}"
+            return False, f"Unexpected exception {err}"
```

Comment:
exception response would be incorrect if exception=None. I would recommend to define this function to make it generic in such case.

Reply:
Addressed